### PR TITLE
Make CICD check the CLI commands

### PR DESCRIPTION
**Pull request by haixuanTao**
This commit improves the CI/CD by testing the newly added CLI commands:
- `up`
- `list`
- `check`
- `destroy`

This commit also removes not needed anymore capnp dependencies to simplify the CI.

This can be merged after `v0.1.0` release that is going to be tagged to #143

### CODE REVIEW

File: examples/python-dataflow/dataflow_without_webcam.yml [hunk 1]
```diff
@@ -5,7 +5,8 @@ communication:
 nodes:
   - id: no_webcam
     custom:
-      source: ./no_webcam.py
+      source: python
```
Comment (phil-opp) [inline examples/python-dataflow/dataflow_without_webcam.yml:8]: Is there any specific reason for that change?

**Review by phil-opp — APPROVED** [review]
Looks good to me! There are still a few CI errors though.

File: examples/python-dataflow/dataflow_without_webcam.yml (hunk 1 above)
Comment (haixuanTao) [reply examples/python-dataflow/dataflow_without_webcam.yml:8]: it's just cosmetics. But I will remove it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -66,16 +66,61 @@ jobs:
...
+          sudo apt-get install -y libacl1-dev
+
+      - name: "Build cli and binaries"
+        timeout-minutes: 30
+        run: |
+          cargo install --path binaries/coordinator
+          cargo install --path binaries/runtime
+          cargo install --path binaries/cli
+
+      - name: "Test dora-coordinator"
```
Comment (phil-opp) [inline .github/workflows/ci.yml:100]:
```suggestion
      - name: "Start dora-coordinator"
```

**Review by phil-opp — APPROVED** [review]
Looks good! Thanks a lot for working through all the errors!

**Pull request merged**